src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -428,7 +459,7 @@ public Participation findOneWithEagerSubmissionsAndResults(Long id) {
      * @return the participation with all its results
      */
     @Transactional(readOnly = true)
-    public Participation findOneWithEagerResultsAndSubmissions(Long id) {
+    public Participation findOneWithEagerResultsAndSubmissionsAndAssessor(Long id) {
```

Comment:
What kind of id do we have here? I guess a participationId. If yes, please call it participationId then to be absolutely clear. (By the way we should do that in all cases in the future and never use id as parameter of a method)

Reply:
You are right, I it to `participationId`. I also adjusted the naming of the `id` parameters of all other methods in this class.